internal/controllers/addon/controller.go

```diff
@@ -173,6 +174,13 @@ func (r *AddonReconciler) InjectOCMClient(ctx context.Context, c *ocm.Client) er
 	return nil
 }
 
+func (r *AddonReconciler) GetOCMClusterID() string {
+	if r.ocmClient == nil {
+		return ""
+	}
+	return r.ocmClient.GetClusterID()
```

Comment:
Needs to guard access via Mutex lock `ocmClientMux`

Reply:
done 👍